sources/platform/console/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Apify Console
+description: Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.
+sidebar_position: 1
+category: platform
+slug: /console
+---
+
+**Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.**
+
+---
+
+## Sign Up
+
+To use Apify Console, you need to create Apify account. [Sign up here](https://console.apify.com/sign-up).
+
+![apify-console-sign-up](./images/console-signup.png)
+
+## Log In
+
+To log in to Apify Console, you can use one of the following methods:
+
+* Google Single Sign-On
+* GitHub Single Sign-On
+* The email and password set during account creation
+
+![apify-console-login](./images/console-login.png)
+
+## Homepage Overview
+
+![apify-console-homepage](./images/console-homepage.png)
+
+The Apify Console homepage provides an overview of our services. The header displays your account name and current plan level. The homepage features several sections:
+
+* **Get Started with Apify**: This section guides you to Apify Store. Here, you can explore existing Actors or create your own. A 'Show me how' button directs you to a guided tour on the Console.
+
+* **Recently Viewed**: This section displays Actors you have recently accessed. New users or those who haven't viewed any Actors will see suggestions instead.
```

Comment:
Note: The way we talk to the reader changes here. Above it's `you can explore`, here it is `new users or those...`

Reply:
Something like `If you are a new user or haven't viewed any Actor yet, you will see suggestions instead` I think that might preserve the consistency of addressing the user. Now that I think about it we might just say `If you haven't used any Actor yet, you will see suggestions instead` because it is obvious that new users would not have used any yet